templates/helm.yml

```diff
@@ -96,11 +96,17 @@
       helm search repo current-repo/"${CHART_NAME}"
       echo
 
+      LATEST=$(helm search repo current-repo/"${CHART_NAME}" | tail -n 1 | awk '{print $2}')
```

Comment:
Sposterei questo direttamente alla linea 102 evitando la variabile in più, bel trick